### DISCUSSION TITLE
`record_property` for xunit2

### DISCUSSION THREAD

**Discussion by timofurrer** [pytest-dev/pytest#9106] [Q&A] [answered]
Is there a way to record additional properties for an `xunit2` format. For `xunit1` there is `record_property`, but it gives a warning when used in combination with `xunit2`:

```
PytestWarning: record_property is incompatible with junit_family 'xunit2' (use 'legacy' or 'xunit1')
```

**Comment by RonnyPfannschmidt** [comment, marked as answer]
@timofurrer no, the junit2 format does only support properties of a testsuite - modeling a test as a single element suite would be a most unacceptable hack
